javascript-client/test/api/ProjectApi.spec.js

```diff
@@ -107,77 +72,65 @@
           ]
         };
 
-        var projectId;
+        instance.apiClient.authentications['Bearer'].apiKey = userToken;
+        instance.apiClient.authentications['Bearer'].apiKeyPrefix = "Bearer";
 
         instance.addProject(project, function (error, data, res) {
           if (error) throw error;
-          projectId = res.body.project._id;
-          instance.projectDetailsIdGet(projectId, function (error, res) {
+          let projectId = res.body.project._id;
+          instance.projectDetailsIdGet(projectId, function (error, data, res) {
+            if (error) throw error;
             expect(res.status).to.be(200);
             done();
           })
-          expect(res.status).to.be(200);
-          done();
         });
+      });
+      it('Admin CRUD successfully', async function (done) {
```

Comment:
Please describe this test better. I assume you mean something like:
'Admin user should be able to CRUD project'